packages/@react-native-windows/virtualized-list/src/VirtualizedList.js

```diff
@@ -458,7 +461,7 @@ class VirtualizedList extends React.PureComponent<Props, State> {
       });
       return;
     }
-    const frame = this._getFrameMetricsApprox(index);
+    const frame = this._getFrameMetricsApprox(index, /*useRawMetrics:*/ true);
```

Comment:
Trying to wrap my head around the coordinate spaces here.

So in the default case, we use lets say a "U-V" coordinate space, where we monotonically increase in position from front to end. This means inverting the coordinate space given from layout events in the cell.

But then for scroll ViewManager commands, we are expecting in native that we have raw uninverted XY coordinates?

Wondering what the iOS/Android coordinate space for `scrollTo` is when inverted. I.e. if we would want to change scroll behavior in native, instead of inverting for VirtualizedList.

Reply:
The original implementation we had for this did invert the scroll offsets in native. This made virtualization entirely impossible, because we can't feasibly invert the `onLayout` events emitted by items, so the scroll viewer used relative coordinates and the onLayout event of items used absolute coordinates.

This is the "least bad" approach IMO. That being said, if I move the metrics out into their own component, we can add a `getRawFrameMetricsApprox` and `getFrameMetricsApprox` or something like that.

In iOS / Android, they do not need to invert coordinate spaces because the list is entirely flipped via a scaleY transform.

Well, sort of. There's actually a weird quirk when you run measure layout in an inverted list on iOS and Android that looks like the following:
![image](https://user-images.githubusercontent.com/1106239/130083191-22b4fe94-3689-43ee-80f4-82a1281b1694.png)
So while the list items are monotonically increasing, the layout of the sub-items runs in the reverse direction 😬